update-images.sh

```diff
@@ -42,6 +42,19 @@ do
 
   tar -xf "${RELEASE_FILENAME}" -C "${VERSION}"
 
+  if jq -e -r '."'$VERSION'"' hotfixes.json > /dev/null; then
+    # Applying hotfix for $VERSION
+    HOTFIX_PATH=$(jq -e -r '."'$VERSION'".hotfix_path' hotfixes.json)
```

Comment:
on line 45, there's no dot between `VERSION'"` and `hotfix.json`. Is that difference correct?

Reply:
Yes, that `.` is for jq it is there in both lines (47 and 45)